web/server/watch/functional_core_test.go

```diff
@@ -123,28 +123,26 @@ func TestParseProfile(t *testing.T) {
 		description    string
 		input          string
 		resultIgnored  bool
+		resultTestTags []string
 		resultTestArgs []string
 	}{
 		{
-			SKIP:           false,
-			description:    "Blank profile",
-			input:          "",
-			resultIgnored:  false,
-			resultTestArgs: []string{},
```

Comment:
profile simply returns nil slices now, instead of zero-length slices. Since all the code uses append, and `foo = append([]string(nil), "thing")` is perfectly valid, these lines just get to go away. Zero-value initializers and all that.